packages/webpack/test/worker/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("//packages/webpack:index.bzl", "webpack")
+load("@build_bazel_rules_nodejs//:index.bzl", "generated_file_test")
+
+webpack(
+    name = "bundle_worker",
+    outs = ["app.bundle.worker.js"],
+    args = [
+        "./$(execpath index.js)",
+        "--mode",
+        "production",
+        # Discuss this: Whether we should automatically add this or not
```

Comment:
Following the example of rollup and esbuild I think this style is great. It allows for the most amount of user customization